gflownet/envs/crystals/composition.py

```diff
@@ -290,64 +373,13 @@ def get_element_mask(min_atoms, max_atoms):
         # If required, refine the masks by doing compatibility checks between
         # the space group and the number of atoms
         if self.do_spacegroup_check and isinstance(self.space_group, int):
-            space_group = get_space_group(self.space_group)
-            n_atoms = [s for s in state if s > 0]
-
-            # Get the greated common divisor of the group's wyckoff position.
-            # It cannot be valid to add a number of atoms that is not a
-            # multiple of this value
-            wyckoff_gcd = space_group_wyckoff_gcd(self.space_group)
-
-            # Get the multiplicity of the group's most specific wyckoff position with
-            # at least one degree of freedom
-            free_multiplicity = space_group_lowest_free_wp_multiplicity(
-                self.space_group
+            (
+                mask_required_element,
+                mask_unrequired_element,
+            ) = self._refine_compatibility_check(
+                state, mask_required_element, mask_unrequired_element
```

Comment:
_refine_compatibility_check doesn't return anything anymore so I'd guess that this doesn't run in practice.

Reply:
Thanks for the catch! Fixed in [d8736db](https://github.com/alexhernandezgarcia/gflownet/pull/165/commits/d8736db677f78dc73e4e6c5361689008b3cbd71b).